fs/config/configflags/configflags.go

```diff
@@ -322,6 +324,12 @@ func SetFlags(ci *fs.ConfigInfo) {
 	multiThreadStreamsFlag := pflag.Lookup("multi-thread-streams")
 	ci.MultiThreadSet = multiThreadStreamsFlag != nil && multiThreadStreamsFlag.Changed
 
+	partialUploadExtension = strings.TrimSpace(partialUploadExtension)
+	if len(partialUploadExtension) < 1 || len(partialUploadExtension) > 15 {
```

Comment:
Just looking at rclone's other options, it would be more consistent to call this `--partial-suffix`

Also upload is inaccurate since these are used for downloads to local, so let's leave it out.

So the default should be `.partial` and an empty string is acceptable.